good-enough-practices-for-scientific-computing.tex

```diff
@@ -434,32 +449,52 @@ \subsection*{Acknowledgments}
 
 \item
   \practice{2b}{Decompose programs into functions} that are no more than
-  one page (about 60 lines) long, do not
-  use global variables (constants are OK), and take no more than half
-  a dozen parameters.  The key motivation here is to fit the program
+  one page (about 60 lines) long. Functions are sections within a program
```

Comment:
A function is a reusable part of program that can be treated as a black box by other parts of the program.